app/Importer/LicenseImporter.php

```diff
@@ -73,12 +73,21 @@ public function createLicenseIfNotExists(array $row)
                 $asset = Asset::where('asset_tag', $asset_tag)->first();
                 $targetLicense = $license->freeSeat();
                 if ($checkout_target) {
```

Comment:
I feel like we should maybe be looking at our `$targetSeat` instead of the `$checkout_target` here? Or maybe both? (e.g. `$checkout_target && $targetSeat`)

Reply:
I take care of this, but I put the checkout_target it in place of the try/catch. I don't like if's inside if's... but I think this is clearer.